c7n/resources/account.py

```diff
@@ -144,8 +144,8 @@ def process(self, resources, event=None):
         if self.data.get('multi-region'):
             trails = [t for t in trails if t.get('IsMultiRegionTrail')]
         if self.data.get('notifies'):
-            trails = [t for t in trails if t.get('SNSTopicArn')]
-        if self.data.get('running', True):
+            trails = [t for t in trails if t.get('SnsTopicARN')]
+        if self.data.get('running'):
```

Comment:
thanks for the pr and notify fix, could you elaborate on why the switch on default 'running' value here? that would imply we're not checking for status on a trail by default.

Reply:
Although I get the point to check by default that the trail is active, this is somewhat misleading from a user perspective and this behaviour is not documented (this led me to failed tests until I found this issue). I think we should let the user specify this option and not infer on what he may want or not...

Reply:
I wouldn't want to change schema defaults unless it was causing a major issue.  At this point the expectation by current users would be that it's defaulted to `True`, changing that would break a lot of integrations with cloud custodian.